maat-court-data-api/src/main/resources/db/changelog/changes/lcam_1662_grant_mla_crud_permissions_on_passport_ass_evidence.sql

```diff
@@ -0,0 +1 @@
+GRANT INSERT, SELECT, UPDATE, DELETE ON TOGDATA.PASSPORT_ASS_EVIDENCE TO MLA;
```

Comment:
Do we just need SELECT privileges at the moment as we are only retrieving passport evidences for the find rep order endpoint?

Reply:
Ok that makes sense, happy to resolve as we need to keep them both in line to make the logic in orchestrator simpler.